pyproject.toml

```diff
@@ -26,7 +26,7 @@ dependencies = [
     # pandas: "excel" is an extra that contains xlrd that is necessary for reading old .xls Excel files
     "pyld>=2.0.4",
     "pyyaml>=6.0.2",
-    "rdflib>=7.1.1",
+    "rdflib==7.1.1",
```

Comment:
The type hints of rdflib 7.1.2 contain some breaking changes. It's perhaps easier to resolve them in a separate PR